bug: check dependencies don't work on Windows

### Node.js version

v16.17.0

### NPM version

8.15.0

### @checkly/cli version

0.4.2

### Steps to reproduce

Create a Browser Check with local file dependencies (not in the same directory), and run `npx checkly test` on Windows. Deploying the project makes the check to have a `\` path separator.

### What is expected?

The check is able to run and resolve the local file dependencies.

### What is actually happening?

The local file dependencies are uploaded with `\` as a path separator rather than `/`. This then causes the check to fail with `Cannot find module './helpers/helper.js'`.

### Any additional comments?

The CLI should convert the paths to use unix path separators before uploading the checks to Checkly.

Relates to https://github.com/checkly/checkly-cli/issues/539 too.

--- a/packages/cli/package.json
+++ b/packages/cli/package.json
@@ -5,9 +5,10 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
+    "clean" : "rimraf ./dist",
     "test": "jest --selectProjects unit",
     "test:e2e": "npm run prepare && NODE_CONFIG_DIR=./e2e/config jest --selectProjects E2E",
-    "prepare": "rm -rf dist && tsc --build",
+    "prepare": "npm run clean && tsc --build",
     "prepack": "npx oclif manifest",
     "lint": "eslint src"
   },

--- a/packages/cli/src/constructs/browser-check.ts
+++ b/packages/cli/src/constructs/browser-check.ts
@@ -96,13 +96,13 @@ export class BrowserCheck extends Check {
     const deps: CheckDependency[] = []
     for (const { filePath, content } of parsed.dependencies) {
       deps.push({
-        path: path.relative(Session.basePath!, filePath),
+        path: path.relative(Session.basePath!, filePath).split(path.sep).join(path.posix.sep),
         content,
       })
     }
     return {
       script: parsed.entrypoint.content,
-      scriptPath: path.relative(Session.basePath!, parsed.entrypoint.filePath),
+      scriptPath: path.relative(Session.basePath!, parsed.entrypoint.filePath).split(path.sep).join(path.posix.sep),
       dependencies: deps,
     }
   }

--- a/packages/create-cli/package.json
+++ b/packages/create-cli/package.json
@@ -7,8 +7,9 @@
   "types": "dist/index.d.ts",
   "private": false,
   "scripts": {
+    "clean" : "rimraf ./dist",
     "test": "jest --selectProjects unit",
-    "prepare": "rm -rf dist && tsc --build",
+    "prepare": "npm run clean && tsc --build",
     "prepack": "npx oclif manifest",
     "lint": "eslint src",
     "start": "node ./index.mjs"

